src/common/menu.rs

```diff
@@ -189,21 +194,23 @@ macro_rules! add_single_option_submenu {
                 MENU.$id as usize,
                 [<$id _toggles>].iter().map(|(x, y)| (x.as_str(), *y)).collect::<Vec<(&str, usize)>>(),
                 [].to_vec(),
-                DEFAULT_MENU.$id as usize
+                DEFAULT_MENU.$id as usize,
+                stringify!($help_text),
             );
         }
     }
 }
 
 macro_rules! add_onoff_submenu {
-    ($menu:ident, $title:literal, $id:ident) => {
+    ($menu:ident, $title:literal, $id:ident, $help_text:literal) => {
         paste::paste!{
             $menu.add_sub_menu_onoff(
                 $title, 
                 stringify!($id), 
                 MENU.$id as usize,
                 (MENU.$id as usize & OnOff::On as usize) != 0,
-                DEFAULT_MENU.$id as usize
+                DEFAULT_MENU.$id as usize,
+                stringify!($help_text),
```

Comment:
`stringify!` is unnecessary here

Reply:
I think it is necessary - when omitted, there is a double quotation issue in the HTML which causes the onfocus events not to execute